Directory.Build.props

```diff
@@ -87,6 +87,11 @@
     <NuspecBasePath>$(MSBuildProjectDirectory)</NuspecBasePath>
 
     <IncludeSymbols>true</IncludeSymbols>
+    <!--
+      Also update the DefaultNetCoreTargetFramework defined at src/Razor/test/testassets/Directory.Build.props and
+      the packages referenced in Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj when this changes.
+    -->
+    <DefaultNetCoreTargetFramework>net6.0</DefaultNetCoreTargetFramework>
```

Comment:
This property is defined at https://github.com/dotnet/aspnetcore/pull/26788/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R40 and should not be duplicated here. Fix that line instead. But, sure, extend the comments there.

Reply:
Can you clarify what you mean here, Doug? The GH diff link doesn't send me to a location where I can parse your statement.

What's the line you're referring to?